.github/workflows/tiiuae-pixhawk-and-saluki.yaml

```diff
@@ -344,3 +344,107 @@ jobs:
           echo "${flash_tool_cmd}" >> $GITHUB_STEP_SUMMARY
           echo '```' >> $GITHUB_STEP_SUMMARY
           echo 'For more info please see: https://github.com/tiiuae/fpga-flashing/#usage' >> $GITHUB_STEP_SUMMARY
+
+  upload-minimal-ota-container:
+    name: upload minimal ota container to docker registry
+    runs-on: ubuntu-latest
+    env:
+      COMBINED_OUTPUT_JSON: ${{ github.workspace }}/repackage/combine.json
+      BUILD_DIR: bin
+    needs:
+      - variables
+      - fc_matrix
+    steps:
+      - name: Checkout px4-firmware
+        uses: actions/checkout@v4
+        with:
+          path: px4-firmware
+          submodules: true
+          token: ${{ secrets.GH_REPO_TOKEN }}
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+        with:
+          platforms: arm64
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Download pixhawk artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: ${{ env.BUILD_DIR }}
+          pattern: saluki-*
+          merge-multiple: true
+      - name: Firmware flasher - Container metadata
+        id: containermeta # referenced from later step
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/tiiuae/px4-firmware-ota-container
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
+            type=sha
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Minimal OTA container
+        run: |
+          set -eux
+          echo "Building minimal OTA container"
+          PRODUCTS_TO_INCLUDE="ssrc_saluki-v3_default ssrc_saluki-v3_flat ssrc_saluki-nxp93_flat ssrc_saluki-nxp93_default"
```

Comment:
Is this enough targets?
- ssrc_saluki-v3_default
- ssrc_saluki-v3_flat
- ssrc_saluki-nxp93_flat
- ssrc_saluki-nxp93_default